src/components/organisms/CesiumViewer.vue

```diff
@@ -1,19 +1,52 @@
 <script lang="ts">
-import { defineComponent, onMounted, ref } from '@vue/composition-api';
+import {
+  defineComponent,
+  onMounted,
+  ref,
+  toRef,
+  watch,
+}
+  from '@vue/composition-api';
 
 import Cesium from '@/plugins/cesium';
 
 export default defineComponent({
   name: 'CesiumViewer',
 
-  setup() {
-    const cesiumViewer = ref(null);
+  props: {
+    location: {
+      type: Object, /* TODO typescript interfaces for props */
+      required: false,
+      default: () => ({}),
```

Comment:
I believe this would be 
```suggestion
      type: PropType<{}>,
      required: false,
      default: () => ({}),
```

Where you can add `PropType` to the existing import from `@vue/composition-api`.

Also, if you know the shape of `location`, I'd suggest defining an interface for it.

Reply:
I believe using`PropType<{}>` here will be more effective once I have the location interface defined ie. `Object as PropType<Location>`.
I have that planned for my next PR as it makes sense to me to group it with other code I am working on.